dev-resources/private/test/src/cljc/foo/bar/core.cljc

```diff
@@ -0,0 +1,16 @@
+(ns foo.bar.core
+  (#?(:clj  :require
+      :cljs :require-macros)
+    [foo.bar.macros]))
```

Comment:
For me is ok, everything one the same line?
`(#?(:clj  :require :cljs :require-macros) [foo.bar.macros]))`
or
```
(#?(:clj  :require
     :cljs :require-macros) [foo.bar.macros]))
```